### ISSUE TITLE
你好，想问一下cpu是12400支持吗

### ISSUE THREAD

**Issue by tanzexing** [vuthanhdanh080200/EFI-i512400F-AsrockZ690ProRs#1]
你好，想问一下cpu是12400支持吗

**Comment by usawarion999** [comment]
Yes, it will work. You can check from [here](https://www.reddit.com/r/hackintosh/comments/sp1zgv/opencore_alder_lake_12thgen_intel_hackintosh/?onetap_auto=true)

**Comment by usawarion999** [comment]
I follow this [post](https://www.tonymacx86.com/threads/asrock-z690-steel-legend-13900kf-12900k-6600xt.317456/) to create EFI.
